recipe/meta.yaml

```diff
@@ -42,10 +42,12 @@ requirements:
     - geopandas
     - cdsapi
     - pytest
+    - cfgrib >=0.9,15.0
```

Comment:
Is there a typo? I think the comma should be a point, i.e. instead of `0.9,15.0` it should be `0.9.15.0`

Reply:
Yes, you are right. Fixed.